### PR TITLE
Update to gym==0.26.x

### PR DESCRIPTION
**Pull request by frederikschubert**
This PR updates the requirements, wrappers and examples to the new API introduced by `gym==0.26.0`. 

Depends on https://github.com/coax-dev/coax/pull/27

### CODE REVIEW

File: requirements.dev.txt [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+jaxlib>=0.3.20
 flake8>=3.8.4
 pylint>=2.6.0
 pur>=5.3.0
 pytest>=6.1.2
 pytest-cov>=2.10.1
 pytest-xdist>=2.1.0
 twine>=3.2.0
-gym[box2d]>=0.21.0
+gym[box2d]>=0.26.0
```
Comment (KristianHolsheimer) [inline requirements.dev.txt:10]: Dependency already covered in the main `requirements.txt`. Please remove.

**Review by KristianHolsheimer — CHANGES_REQUESTED** [review]

**Comment by KristianHolsheimer** [comment]
There seems to be a lot of overlap with #27. Do we need both PRs?

**Review by KristianHolsheimer — CHANGES_REQUESTED** [review]
Thanks so much for this!

Please could you update the examples in the docs as well?

For instance: [`doc/index.rst`](https://github.com/coax-dev/coax/blob/main/doc/index.rst)

**Comment by frederikschubert** [comment]
> There seems to be a lot of overlap with #27. Do we need both PRs?

Yes, I basically wanted to show the changes separately. In the end, we can just merge https://github.com/coax-dev/coax/pull/29 as it contains all other changes as well.

**Review by KristianHolsheimer — APPROVED** [review]

**Pull request merged**